src/main/java/iroha/validation/utils/ValidationUtils.java

```diff
@@ -1,6 +1,7 @@
 package iroha.validation.utils;
 
 import iroha.protocol.TransactionOuterClass.Transaction;
+import jp.co.soramitsu.iroha.java.Utils;
 
 public class ValidationUtils {
```

Comment:
I think we can turn it into an interface.

Reply:
Done